src/Build.UnitTests/BackEnd/LoggingServicesLogMethod_Tests.cs

```diff
@@ -1089,7 +1090,7 @@ public void TaskStartedNullBuildEventContext()
             Assert.Throws<InternalErrorException>(() =>
             {
                 ProcessBuildEventHelper service = (ProcessBuildEventHelper)ProcessBuildEventHelper.CreateLoggingService(LoggerMode.Synchronous, 1);
-                service.LogTaskStarted(null, "MyTask", "ProjectFile", "ProjectFileOfTask");
+                service.LogTaskStarted(null, "MyTask", "ProjectFile", "ProjectFileOfTask", null);
```

Comment:
when passing literals such as `true`, `false` or `null` it's good to add the argument name